script/release/uploaders/upload.py

```diff
@@ -95,6 +97,20 @@ def main():
     shutil.copy2(os.path.join(OUT_DIR, 'debug.zip'), debug_zip)
     upload_electron(release, debug_zip, args)
 
+    # Upload libcxx_objects.zip for linux only
+    libcxx_objects = get_zip_name('libcxx_objects', ELECTRON_VERSION)
+    libcxx_objects_zip = os.path.join(OUT_DIR, libcxx_objects)
+    shutil.copy2(os.path.join(OUT_DIR, 'libcxx_objects.zip'), libcxx_objects_zip)
+    upload_electron(release, libcxx_objects_zip, args)
```

Comment:
In order to upload these as part of the release we'll need to generate them too.  We should add some extra `ninja` calls to our publish steps in the CircleCI config.

Reply:
Added here: https://github.com/electron/electron/pull/29281/commits/421a56c081cf2147999b672428f41b758fd740d8#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R806-R815